os/net/netdev/netdev_ioctl.c

```diff
@@ -416,6 +416,7 @@ static int netdev_getconf(FAR struct netif *dev, void *arg)
 	}
 
 	strncpy(ifr->ifr_name, dev->d_ifname, IFNAMSIZ - 1);
+	ifr->ifr_name[IFNAMSIZ-1] = '\0';
 	struct sockaddr_in *sin = (struct sockaddr_in *)&ifr->ifr_addr;
```

Comment:
this modification doesn't relate to network manager. this file is not compiled network manager is enabled furthermore adding 0 to IFNAMESIZ -1 seems not good.  actually dev->d_ifname should contains 0 character.